lib/dl_core_testing/dl_core_testing/testcases/data_source.py

```diff
@@ -105,6 +105,12 @@ def test_get_raw_schema(
         print("schema:")
         for col in simplified_schema:
             print(col)
+
+        # TODO: This is a workaround for YDB because raw schema is sorted different. Remove it when the issue is fixed.
```

Comment:
Is there a plan on how to fix this or we don't yet even know where the issue lies?